INSTEXE missing .sh file ending

Cactooz opened the pull request:
Creating a noip-renew-$USER.sh file instead of noip-renew-$USER file

Cactooz committed:
INSTEXE missing .sh file ending

Creating a noip-renew-$USER file instead of noip-renew-$USER.sh file

IDemixI commented:
Hi there,

I’m pretty sure this was intended so you could call the script using the “command” noip-renew-pi or whatever as opposed to with the .sh

hugobacher commented:
I'm not so sure, you can still run the script the same way. Just that .sh is at the end. .sh shows that it's a shell file, compared to no file ending, then you don't know which type of file it is.
If you check the default files before the user name gets added ([Check here](https://github.com/loblab/noip-renew/blob/master/noip-renew.sh)) the file actually has the .sh file ending.

Cactooz committed:
Cronjob run at any day if it fails

Setting both day and month to * instead of 0, letting the cronjob try to execute everyday instead of never.

Cactooz committed:
Update due to changes on no-ip.com

Cactooz committed:
Added crontab explanation & Changed crontab time

Cactooz committed:
Merge branch 'master' into patch-1

loblab committed:
Merge branch 'master' into patch-1
